src/mixins/removeHtmlTruncate.js

```diff
@@ -8,18 +11,15 @@ export default {
     methods: {
         removeHtmlTruncate(str = "", maxlength = "") {
             // Remove HTML
-            let stripHtml = str.replace(/(<([^>]+)>)/gi, "")
-            let removeQuotes = stripHtml.replace(/"/gi, "")
-            console.log("this is maxlength" + maxlength)
-            if (str.length < maxlength || maxlength == "") {
-                return removeQuotes
-            } else {
-                // Truncate
-                let limitCharacter = removeQuotes.substring(0, maxlength)
-                let words = limitCharacter.split(" ")
+            let stripHtml = _unescape(
```

Comment:
We need to keep the logic that just returns the unescaped string if there is no maxlength provided. See this story https://deploy-preview-297--ucla-library-storybook.netlify.app/?path=/story/block-card-with-illustration--databases-visit

Reply:
Oh, good catch. I shouldn't have stuck with "" as the default value, since javascript treats it as 0. Fortunately javascript has a built in `Infinity`